### PR TITLE
RDCC-693 Raising PR for blob store file transfer functionality

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java [hunk 1]
```diff
@@ -17,8 +43,108 @@
         "uk.gov.hmcts.reform.professionalapi",
 })
 @SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
-public class Application  {
+public class Application  implements CommandLineRunner {
+
+    private static final String BLOB_URL = "http://3fbfcfb8181346cabaf7.blob.core.windows.net";
+    private static final String BLOB_HTTPS_URL = "https://3fbfcfb8181346cabaf7.blob.core.windows.net";
+    private static final boolean isEnableHttps = false;
+    private static final String ASURE_ACCOUNT_NAME = "3fbfcfb8181346cabaf7";
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: change Asure to Azure :)

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java [hunk 2]
```diff
@@ -17,8 +43,108 @@
...
+        ServiceURL serviceUrl = createServiceUrl(new PipelineOptions());
+        ContainerURL containerUrl = serviceUrl.createContainerURL(CONTAINER_NAME);
+        createContainer(containerUrl);
+        final BlockBlobURL blockBlobUrl = containerUrl.createBlockBlobURL(sourceCsvFile);
+        uploadFile(blockBlobUrl, sourceFile);
+    }
+
+    @SuppressWarnings("deprecation")
+    public static void uploadFile(BlockBlobURL blob, File sourceFile) throws IOException {
+        log.info("Start uploading file %s...", sourceFile);
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: why we are using deprectaed methods/code?

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java [hunk 3]
```diff
@@ -17,8 +43,108 @@
...
+        }, 0, 60 * 1000 * 10);
+    }
+
+    public void pushFile() throws IOException, InvalidKeyException {
+
+        final String sourceCsvFile = "content.csv";
+        final File sourceFile = new File(this.getClass().getClassLoader().getResource(sourceCsvFile).getFile());
+        ServiceURL serviceUrl = createServiceUrl(new PipelineOptions());
+        ContainerURL containerUrl = serviceUrl.createContainerURL(CONTAINER_NAME);
+        createContainer(containerUrl);
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: do we need to create container?

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java (hunk 2 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: Replaced with ignoreElements()

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java (hunk 3 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: not required and removed

File: src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java]: Renamed.

**Pull request closed without merging**